util/mk1mf.pl

```diff
@@ -304,6 +304,7 @@
 $cflags.=" -DOPENSSL_NO_ASYNC" if $no_async;
 $cflags.=" -DOPENSSL_NO_AUTOALGINIT" if $no_autoalginit;
 $cflags.=" -DOPENSSL_NO_AUTOERRINIT" if $no_autoerrinit;
+$cflags.=" -DOPENSSL_NO_NEXTPROTONEG" if $no_nextprotoneg;
```

Comment:
I do not think this change is required as this is definition gets generated in include/openssl/opensslconf.h